### ISSUE TITLE
Embed Google Material Design Icons

### ISSUE THREAD

**Issue by dlabordus** [com-pas/compas-open-scd#41]
As a developer
I want to included the Google Icons in the source
So that we don't have a dependency with Google at runtime

Background:
We currently retrieve the Icons from https://fonts.googleapis.com/css?family=Material+Icons+Outlined&display=block
In some environments that isn't allowed, because it's a external source.

Some info to check is 
http://google.github.io/material-design-icons/#icon-font-for-the-web
https://github.com/google/material-design-icons/tree/master/font

**Comment by Sander3003** [comment]
@ca-d is this some thing to summit upstream to OpenSCD?

**Comment by ca-d** [comment]
> @ca-d is this some thing to summit upstream to OpenSCD?

Absolutely, I hadn't seen this pull request yet, thanks for the mention! We were simply lazy not do bundle both these icons and the `Roboto` fonts we rely on from the start. We'd be more than happy to accept any work to that effect. These webfonts currently constitute our only reliance on an internet connection at install time as far as I know.

**Comment by Sander3003** [comment]
This is currently in the idea phase, we planning to work on this soon.
